docs/spectrum1d.rst

```diff
@@ -47,7 +47,22 @@ encouraged to :doc:`create their own loader </custom_loading>`.
     >>> from specutils import Spectrum1D
     >>> spec1d = Spectrum1D.read("/path/to/file.fits")  # doctest: +SKIP
 
+Most of the built-in specutils default loaders can also read an existing 
+`astropy.io.fits.HDUList` object or an open file object (as resulting 
+from e.g. streaming a file from the internet). Note that in these cases, a 
+format string corresponding to an existing loader must be supplied.
 
+.. code-block:: python
+    
+    >>> from astroquery.sdss import SDSS # doctest: +SKIP
```

Comment:
It might be better to separate the `astroquery` and `urllib` into to separate blocks.  The complication being that `astroquery` requires the user to have `astroquery` installed.  So you might have this be a second example with some explanatory text like "or to download the same spectrum using `astroquery`, you could do:" so that the user knows that's a more convenient way to do the same thing?